### PR TITLE
Translation-ja: Add trusted-committer/ja/06-advocating-for-the-communitys-needs-ja

### PR DESCRIPTION
**Pull request by ystk**
This PR is a part of #367 .

### CODE REVIEW

File: trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Trusted Committerが重要な役割を果たすのは、この潜在的な競合領域です。
+Trusted Committerは、組織との信頼関係を構築し、その信頼関係に基づいて、コミュニティの利益と企業内のソフトウェアの長期的な健全性の擁護者として行動します。
+彼らには、技術的なリスクとコミュニティ関連のリスクを経営層に伝える責任があります。
+同時に、Trusted Committerは戦略的かつ会社により与えられる自由度の範囲内で活動する必要があります。
+
+また、Trusted Committersは、コミュニティと個々の  https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[コントリビューター] が、彼らの仕事に対する公な称賛を得られるようにする必要があります。
+公な称賛とは、特に自発的にコントリビューションするコントリビューターに与えられ広まるものです。
+有益なコントリビューターを公に称え、彼らのマネージャーがコントリビューションを認識していることを確認することは良い慣行です。
+称賛を与えることを怠ると、個々のコントリビューターを苛立たせ、コミュニティの健康を損ねることにもなりかねません。
```
Comment (johtani) [inline trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc:20]:
```suggestion
称賛を与えることを怠ると、個々のコントリビューターを苛立たせ、コミュニティの健全性を損ねることにもなりかねません。
```

File: trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+これは、InnerSourceの作業モデルにまだ慣れていない企業や、陰で動いているInnerSourceコミュニティによってソフトウェアが開発されている場合や、マネージャが単にコミュニティの貢献に気付いていなかった場合に起こります。
+優れたTrusted Committerは、マネージメント層と連携して公の称賛を唱道します。
```
Comment (johtani) [inline trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc:22]:
```suggestion
優れたTrusted Committerは、マネージメント層と連携して公の称賛を主張します。
```

File: trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+称賛を与えないことは、悪意で行われることはほとんどなく、簡単に修正できます。
+
+Trusted Committerによる擁護が必要となるもう1つの一般的なケースは、 https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[コントリビューター] に投稿する時間や許可が与えられていない場合です
```
Comment (johtani) [inline trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc:25]:
```suggestion
Trusted Committerによる支持が必要となるもう1つの一般的なケースは、 https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[コントリビューター] にコントリビューションする時間や許可が与えられていない場合です
```

File: trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+これは、コミュニティがコントリビューターの部署外で製品を開発している場合で、マネージャの目標との関係がない場合に起こる可能性があります。
+この場合、Trusted Committer はコントリビュータのマネージャと議論し、代替案の決定を求めロビー活動を行う必要があります。
```
Comment (johtani) [inline trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc:27]:
```suggestion
この場合、Trusted Committer はコントリビュータのマネージャと議論し、代替案の決定を求め働きかける必要があります。
```

File: trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+要約すると、Trusted Committerが、個々の https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[コントリビューター] の利益とコミュニティ全体を擁護する必要がある状況は多々あります。
```
Comment (johtani) [inline trusted-committer/ja/06-advocating-for-the-communitys-needs-ja.asciidoc:29]:
```suggestion
要約すると、Trusted Committerが、個々の https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[コントリビューター] とコミュニティ全体の利益を主張する必要がある状況は多々あります。
```
「両方の利益を主張する」というほうが意味が通るかな？という気がしたので修正してみました。

**Review by johtani — COMMENTED** [review]
Great work, @ystk .
I left some small suggestions, these were not mandatory.
Please review my suggestions and approve these if you like.

**Comment by ystk** [comment]
I was wondering how to pick up right words for "advocate" in Japanese. I agree with your suggestions. Thanks a lot @johtani !

**Review by johtani — APPROVED** [review]
LGTM! Ghanks, @ystk

**Comment by rrrutledge** [comment]
Nice work, everyone!

**Comment by rrrutledge** [comment]
Opened a [pull request](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/123) to put them on innersourcecommons.org.

**Pull request merged**